clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/lifecycle/LaunchFailureConfigurationProperties.groovy

```diff
@@ -0,0 +1,49 @@
+package com.netflix.spinnaker.clouddriver.aws.lifecycle
+
+import groovy.transform.Canonical
+import org.springframework.boot.context.properties.ConfigurationProperties
+
+/*
+ * Copyright 2017 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@ConfigurationProperties("aws.lifecycleSubscribers.launchFailure")
+class LaunchFailureConfigurationProperties {
+  String accountName
+  String topicARN
+  String queueARN
+
+  int maxMessagesPerCycle = 1000
```

Comment:
Idea being that every time the subscriber runs ... it'll read at most `maxMessagesPerCycle` from the SQS queue.